m_BVM_p4.R

```diff
@@ -1539,7 +1540,7 @@ MA_sroc_plot_server <- function(id,
       # Download ggplot object 
       output$plot_download <- downloadHandler(
         filename = function() {
-          paste("plot.png")
+          paste0("plot", input$plot_file_type)
```

Comment:
It would be nice to have a more descriptive file name (like we are trying to do for MetaInsight)